### DISCUSSION TITLE
Impact of common ice mask in surface mass balance estimates over the Antarctic ice sheet

### DISCUSSION THREAD

**Discussion by mankoff** [IACS-cryo/Delineation-WG#25] [Paper discussions]
I'm surprised we had not discussed this paper, nor have any of these authors as part of this effort. I will email them to try to repair this.

https://doi.org/10.5194/tc-16-711-2022

Abstract

> Regional climate models compute ice sheet surface mass balance (SMB) over a mask that defines the area covered by glacier ice, but ice masks have not been harmonised between models. Intercomparison studies of modelled SMB therefore use a common ice mask. The SMB in areas outside the common ice mask, which are typically coastal and high-precipitation regions, is discarded. Ice mask differences change integrated SMB by between 40.5 and 140.6 Gt yr−1 (1.8 % to 6.0 % of ensemble mean SMB), equivalent to the entire Antarctic mass imbalance. We conclude there is a pressing need for a common ice mask protocol.

This appears to be an outcome of https://doi.org/10.5194/tc-15-3751-2021 "What is the surface mass balance of Antarctica? An intercomparison of regional climate model estimates"

**Comment by NicolajHansen90** [comment]
Hi @mankoff 
I am glad that you found our paper interesting.
Thanks for the meeting last week, I enjoyed it very much.
Regarding what we talked about at GUES, this delineation WG from the RCM point of view. Should I start a "discussion" or an "issue"? Or how do you prefer it? (I am not that used to using GitHub as a communication challenge, so I do not know the best way to do it)

**Reply by NicolajHansen90** [reply]
I guess that it should be an issue based on this 
https://github.com/IACS-cryo/Delineation-WG/issues/12

**Reply by mankoff** [reply]
Hi @NicolajHansen90 - we're also figuring out the best way to use GitHub. 

I don't have strong opinions on how to use it, as long as it (or some other open communication system) is used. 

I think the distinction between Discussion and Issue is well-framed in #12, but I note it's still a fuzzy boundary, and I don't know how general or specific you are currently thinking about the RCM component. It is also possible that Issue-level work occurs elsewhere - preferably not in personal, but maybe other-project repositories.

I spoke with a CISM developer at the EGU Splinter Meeting and it seems that it may not be that computationally expensive to update that model to a new boundary. Perhaps while we discuss inter-model it's a discussion, and when it gets to intra-model specifics, we move to Issue?

**Reply by NicolajHansen90** [reply]
Okay, let us do that.
However, even though they say that rerunning CISM is not computationally expensive, we still have all the RCMs. Also, even though it is not expensive, it still requires man-hours to do, new versions of datasets etc. So I still do not think that it is straightforward to do/implement.
